client/src/App.js

```diff
@@ -11,7 +11,7 @@ const App = () => {
       <Header />
       <Switch>
         <Route exact path="/auth" render={() => <Auth />} />
-        <Route exact path="/about-us" render={() => <AboutUs />} />
+        <Route exact path="/AboutUs" render={() => <AboutUs />} />
```

Comment:
will change back in future PR -- routes should be lowercase with words separated by dashes